mne/channels/channels.py

```diff
@@ -1072,6 +1072,26 @@ def add_channels(self, add_list, force_update_info=False):
             assert all(len(r) == self.info['nchan'] for r in self._read_picks)
         return self
 
+    def add_reference_channels(self, ref_channels):
```

Comment:
```suggestion
    @fill_doc
    def add_reference_channels(self, ref_channels):
```

Reply:
(Also put that below in the other file!)

Reply:
this will also need to be imported at the top of the file:
`from ..utils import fill_doc`